cmd/simulator/load/loader.go

```diff
@@ -129,5 +135,67 @@ func ExecuteLoader(ctx context.Context, config config.Config) error {
 		return err
 	}
 	log.Info("Tx agents completed successfully.")
+
+	logOtherMetrics(config.Metrics, config.BlockchainIDStr)
 	return nil
 }
+
+func logOtherMetrics(metrics string, blockchainIDStr string) error {
+	if metrics == "" {
+		log.Info("unable to get metrics: metrics is empty")
+		return nil
+	}
+
+	if blockchainIDStr == "" {
+		log.Info("unable to get metrics: blockchainIDStr is empty")
+		return nil
+	}
+
+	getCallStart := time.Now()
+	resp, err := http.Get(metrics)
+	getCallDuration := time.Since(getCallStart)
+
+	log.Info("GET Metrics API Data", "time", getCallDuration.Seconds())
+	if err != nil {
+		return fmt.Errorf("failed getting metrics: %w", err)
+	}
+
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return fmt.Errorf("failed reading response body of metrics: %w", err)
+	}
+
+	bodyString := string(body)
+
+	buildBlockTime, err := findMatchFromString(bodyString, fmt.Sprintf(".*avalanche_%s_vm_metervm_build_block_sum.*", blockchainIDStr))
+	if err != nil {
+		log.Info("No buildBlock metrics found from metrics API for blockchainIDStr", "blockchainIDStr", blockchainIDStr, "metric", metrics)
+		return nil
+	}
+	log.Info("Sum of time (in ns) of a build_block", "time", buildBlockTime)
```

Comment:
why do we return early if we're missing this metric instead of moving on to the rest?

Do we expect to ever fail to find this metric? Should we still log everything we can or error if we do encounter an unexpected error?

Reply:
Yah we don't need to return. In fact, I don't think we should err anywhere in this function and we should just log it. Doesn't make sense to fail the simulation if we mess up metrics